pkg/label/label.go

```diff
@@ -3,6 +3,13 @@
 package label
 
 const (
+	// App label is used to identify Kubernetes resources.
+	App = "app"
+
+	// AppKubernetesVersion label is used to identify the version of Kubernetes
+	// resources.
+	AppKubernetesVersion = "app.kubernetes.io/version"
```

Comment:
We can start putting labels and annotations into `apiextensions`. See https://github.com/giantswarm/apiextensions/tree/master/pkg/label.

Reply:
Cool but I'll do this as a follow up.